apps/angular/src/app/anime-dashboard/anime-dashboard.component.html

```diff
@@ -0,0 +1,44 @@
+<table mat-table [dataSource]="(allAnime$ | async) ?? []">
+	<!-- Image -->
+	<ng-container matColumnDef="imageSourceURL">
+		<th mat-header-cell *matHeaderCellDef>Image</th>
+		<td mat-cell *matCellDef="let element">
+			<div class="image-container">
+				<img [src]="element.imageSourceURL" [alt]="element.titleEnglish"/>
+			</div>
+		</td>
+	</ng-container>
+
+	<!-- English title -->
+	<ng-container matColumnDef="titleEnglish">
+		<th mat-header-cell *matHeaderCellDef>Title (English)</th>
+		<td mat-cell *matCellDef="let element"> {{element.titleEnglish}} </td>
+	</ng-container>
+
+	<!-- Japan title -->
+	<ng-container matColumnDef="titleJapan">
+		<th mat-header-cell *matHeaderCellDef>Title (Japan)</th>
```

Comment:
I believe that the column is related to the title's language. In this case, `Japan` is the name of the country, not of the language. It would be more appropriate to use `Japanese` instead. We're following the same logic with the English title, where we declare the language name but not the country (England).
```suggestion
		<th mat-header-cell *matHeaderCellDef>Title (Japanese)</th>
```